src/oss/confluent.sh

```diff
@@ -686,6 +755,43 @@ config_service() {
     fi
 }
 
+enable_metrics_reporter() {
```

Comment:
Will `enable_metrics_reporter` be run every single time confluent cli starts services?  If yes, then would the echo statements below be redundant if it's run more than once?  ie, the properties file will have each line written to it each time?

Reply:
The properties under `etc` are the source of truth and are not overwritten. 

The properties under `$CONFLUENT_CURRENT` are the ones that are used to start the services with minor changes/additions to the equivalent file in `etc` (such as the one introduced here) and are overwritten completely with every run. 

So there's no problem with re-runs. A new file is created every time.